app/javascript/react/screens/App/Overview/components/Migrations/MigrationsConstants.js

```diff
@@ -0,0 +1,14 @@
+export const MIGRATIONS_NOT_STARTED_SORT_FIELDS = [
+  { id: 'name', title: __('Name'), isNumeric: false },
+  {
+    id: 'configVmLength',
+    title: __('Infrastructure Mapping'),
```

Comment:
should this be removed as a sort type? I don't know that we show the Infra Mapping name on this list view (so sort seems to have no effect).

![infra-mapping-sort](https://user-images.githubusercontent.com/4237045/42650894-b28fe79e-85db-11e8-976d-35921c44d743.gif)

Reply:
Label as per ux guidance @vconzola requested this.

Reply:
Also, this sorts by number of VMs, check the gif in original pr comment

Reply:
ok - sorry, the label name just threw me...